web/index.html

```diff
@@ -1435,18 +1435,29 @@ <h4 class="modal-title" id="switchRegistryModalLabel">Switch Netdata Registry Id
 		colors: '#FF5588',
 		heads: [
 			gaugeChart('Utilization', '12%', '', '#FF5588')
-		]
+		],
+		info: 'Percent of time the disk was doing something. Read from field 10.'
 	},
 
 	'disk.backlog': {
-		colors: '#0099CC'
+		colors: '#0099CC',
+		info: 'Milliseconds spent doing I/Os multiplied by the number of operations currently in progress. Read from field 11.'
```

Comment:
I suggest the same text from kernel documentation:

Backlog is an indication of the duration of pending disk operations. So, while the disk is busy the Linux kernel is multiplying the time spent doing I/O with the number of pending operations. While not accurate, this metric can provide an indication of the expected completion time of the operations in progress.